internal/renderers/services.go

```diff
@@ -65,7 +65,7 @@ func renderServices(f *excelize.File, data ReportData) {
 			setHyperLink(f, "Services", 12, currentRow)
 		}
 
-		configureSheet(f, "Services", heathers, currentRow)
```

Comment:
Ohhhhhh dude I knew there was something wrong here!!!! Thanks for fixing this!

Reply:
It was nothing terrible, since it worked great and wasn't exposed to the end user anyways.